### ISSUE TITLE
Fail to perform many index operations in a single API call

### ISSUE THREAD

**Issue by NITEMAN** [elastic/packetbeat#156]
Using 1.0-beta, outputing directly to Elasticsearch. We see in syslog quite often (2 or 3 times per hour):

~~~
Jun  7 10:00:04 xxxx /usr/bin/packetbeat[47277]: output.go:116: Fail to perform many index operations in a single API call: Post http://packetbeat.our.domain.tld:80/_bulk: write tcp XX.XX.XX.XX:80: broken pipe
~~~
or
~~~
Jun  7 10:00:04 xxxx /usr/bin/packetbeat[47277]: output.go:116: Fail to perform many index operations in a single API call: Post http://packetbeat.infra.axelspringer.es:80/_bulk: EOF
~~~

**Comment by monicasarbu** [comment]
I think the warnings appear when the connection with Elasticsearch is lost. I managed to get similar warnings when closing the connection to Elasticsearch by just killing the Elasticsearch process. 
We should consider adding a re-trying mechanism in order to deal with temporary network issues or connection drops.

**Comment by tsg** [comment]
This should be closed by the ES multi-node support from 1.0.0-beta2. Please let us know if you still see this with beta2. Thanks!

**Comment by NITEMAN** [comment]
Still see similar errors with beta2.

Package information:
~~~
root@host:/home/sbitmedia# dpkg -s packetbeat
Package: packetbeat
Status: install ok installed
Priority: optional
Section: net
Installed-Size: 7748
Maintainer: Monica Sarbu <monica@packetbeat.com>
Architecture: amd64
Version: 1.0.0~beta2
Depends: libc6 (>= 2.2.5), libpcap0.8 (>= 1.0.0)
Conffiles:
 /etc/init.d/packetbeat 2fa84344af3bcd279aa8e44c194ec631
 /etc/packetbeat/packetbeat.yml b0fbca49dc18dbbfe97d18c40cb93717
 /etc/packetbeat/packetbeat.template.json 9ddd60b5ef476b384fa773af88d55761
 /etc/packetbeat/packetbeat.conf 1e4815693508e7d296d553bfa273a054 obsolete
Description: Packetbeat Agent
Homepage: http://packetbeat.com
~~~

Log messages:
~~~
Aug  6 14:02:07 host /usr/bin/packetbeat[13814]: output.go:133: Fail to perform many index operations in a single API call: PerformRequest fails: 400 Bad Request
~~~

I think @monicasarbu might be right and this could be related with communication failures, although the error logged is not descriptive.
